go/host/host.go

```diff
@@ -650,15 +650,20 @@ func (h *host) signAndBroadcastL1Tx(tx types.TxData, tries uint64) error {
 func (h *host) watchForReceipt(txHash common.TxHash) {
 	var receipt *types.Receipt
 	var err error
-	err = retry.Do(func() error {
-		receipt, err = h.ethClient.TransactionReceipt(txHash)
-		return err
-	}, retry.NewTimeoutStrategy(maxWaitForL1Receipt, retryIntervalForL1Receipt))
+	err = retry.Do(
```

Comment:
Minor (subjective) thing but I preferred the formatting as it was before tbh.

When I see functions as parameters in Go code it usually seems to be like that, with the anonymous func started inline to reduce the indentation, like javascript-style (e.g. the `sub.Receive()` example in the [pubsub callback Go docs](https://pkg.go.dev/cloud.google.com/go/pubsub)).

Reply:
Eh, looking around a bit more I see both styles haha. It's probably just what I'm used to from JS.

Reply:
I typically put one thing per line.. I think this is a job for linter :)